Jenkinsfile_CNP

```diff
@@ -120,6 +121,7 @@ withPipeline(type, product, component) {
         }
         env.DEFINITION_STORE_HOST      = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
         env.ELASTIC_SEARCH_FTA_ENABLED = "true"
+        env.DATA_SETUP_ENVIRONMENT = environmentOfDependencies
```

Comment:
we shouldn't need this kind of an env var. I guess this is just left from previous spike.

Reply:
removed